### PR TITLE
docs: add embedding page

### PR DESCRIPTION
**Pull request by bwanglzu**
Add docs for embedding model page.

---

- [ ] This PR references an open issue
- [x] I have added a line about this change to CHANGELOG

### CODE REVIEW

File: docs/get-started/pretrained.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+(pretrained-models)=
+# {octicon}`rocket` Jina Embeddings
+
+Starting with Finetuner 0.7.9,
+we introduce a suite of pre-trained text embedding models licensed under Apache 2.0.
+The model has a range of use cases, including information retrieval, semantic textual similarity, text reranking, and more.
```
Comment (gmastrapas) [inline docs/get-started/pretrained.md:6]:
```suggestion
The models have a range of use cases, including information retrieval, semantic textual similarity, text reranking, and more.
```

File: docs/get-started/pretrained.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+embeddings = finetuner.encode(
+    model=model,
+    data=['how is the weather today', 'What is the current weather like today?']
+)
+print(finetuner.cos_sim(embeddings[0], embeddings[1]))
+```
+
+## Training Data
+
+Jina embedding models is a suit of language models that has been trained using Jina AI's Linnaeus-Clean dataset.
```
Comment (gmastrapas) [inline docs/get-started/pretrained.md:28]:
```suggestion
Jina embedding models is a suit of language models that have been trained using Jina AI's Linnaeus-Clean dataset.
```

**Review by gmastrapas — APPROVED** [review]

File: docs/get-started/pretrained.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+(pretrained-models)=
+# {octicon}`rocket` Jina Embeddings
+
+Starting with Finetuner 0.7.9,
+we introduce a suite of pre-trained text embedding models licensed under Apache 2.0.
```
Comment (scott-martens) [inline docs/get-started/pretrained.md:5]:
```suggestion
we have introduced a suite of pre-trained text embedding models licensed under Apache 2.0.
```

File: docs/get-started/pretrained.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+The model have a range of use cases, including information retrieval, semantic textual similarity, text reranking, and more.
```
Comment (scott-martens) [inline docs/get-started/pretrained.md:6]:
```suggestion
These models have a variety of use cases, including information retrieval, semantic textual similarity, text reranking, and more.
```

File: docs/get-started/pretrained.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+The suite includes the following:
```
Comment (scott-martens) [inline docs/get-started/pretrained.md:7]:
```suggestion
The suite consists of the following models:
```

File: docs/get-started/pretrained.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+- `jina-embedding-s-en-v1` **[Huggingface](jinaai/jina-embedding-s-en-v1)**: With a compact size of just 35 million parameters, the model enables lightning-fast inference while still delivering impressive performance.
```
Comment (scott-martens) [inline docs/get-started/pretrained.md:9]:
```suggestion
- `jina-embedding-s-en-v1` **[Huggingface](jinaai/jina-embedding-s-en-v1)**: This is a compact model with just 35 million parameters, that performs lightning-fast inference while delivering state-of-the-art performance.
```

File: docs/get-started/pretrained.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+- `jina-embedding-b-en-v1` **[Huggingface](jinaai/jina-embedding-b-en-v1)**: With a standard size of 110 million parameters, the model enables fast inference while delivering better performance than our small model.
```
Comment (scott-martens) [inline docs/get-started/pretrained.md:10]:
```suggestion
- `jina-embedding-b-en-v1` **[Huggingface](jinaai/jina-embedding-b-en-v1)**: This model has a size of 110 million parameters, performs fast inference and delivers better performance than our smaller model.
```

File: docs/get-started/pretrained.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+- `jina-embedding-l-en-v1` **[Huggingface](jinaai/jina-embedding-l-en-v1)**: With a size of 330 million parameters, the model enables single-gpu inference while delivering better performance than our small and base model.
```
Comment (scott-martens) [inline docs/get-started/pretrained.md:11]:
```suggestion
- `jina-embedding-l-en-v1` **[Huggingface](jinaai/jina-embedding-l-en-v1)**: This is a relatively large model with a size of 330 million parameters, that performs single-gpu inference and delivers better performance than our other model.
```

File: docs/get-started/pretrained.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+    model=model,
+    data=['how is the weather today', 'What is the current weather like today?']
+)
+print(finetuner.cos_sim(embeddings[0], embeddings[1]))
+```
+
+## Training Data
+
+Jina embedding models is a suit of language models that have been trained using Jina AI's Linnaeus-Clean dataset.
+This dataset consists of 380 million pairs of sentences, which include both query-document pairs.
```
Comment (scott-martens) [inline docs/get-started/pretrained.md:29]:
```suggestion
This dataset consists of 380 million query-document pairs of sentences.
```

File: docs/get-started/pretrained.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+These pairs were obtained from various domains and were carefully selected through a thorough cleaning process.
+The Linnaeus-Full dataset, from which the Linnaeus-Clean dataset is derived, originally contained 1.6 billion sentence pairs.
+
+## Characteristics
+
+Each Jina embedding model allows encoding of up to 512 tokens,
```
Comment (scott-martens) [inline docs/get-started/pretrained.md:35]:
```suggestion
Each Jina embedding model can encode up to 512 tokens,
```

File: docs/get-started/pretrained.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+with any additional tokens being truncated.
```
Comment (scott-martens) [inline docs/get-started/pretrained.md:36]:
```suggestion
with any further tokens being truncated.
```

File: docs/get-started/pretrained.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+The output dimensionality varies across different models.
+Please consult the table below for more details.
```
Comment (scott-martens) [inline docs/get-started/pretrained.md:38]:
```suggestion
The models have different output dimensionalities, as shown in the table below:
```

File: docs/get-started/pretrained.md (hunk 6 above)
Comment (bwanglzu) [reply docs/get-started/pretrained.md:9]: i'll apply the rest but not SOTA, because it;s not the top one

**Review by scott-martens — APPROVED** [review]
LGTM

**Pull request merged**